spec/shared/context/lead_provider_profiles_and_courses.rb

```diff
@@ -11,7 +11,7 @@
   let(:ecf_lead_provider) { cpd_lead_provider.lead_provider }
   let!(:ect_profile) { create(:ect_participant_profile, schedule: default_schedule) }
   let!(:mentor_profile) { create(:mentor_participant_profile, school_cohort: ect_profile.school_cohort, schedule: default_schedule) }
-  let(:induction_coordinator_profile) { create(:induction_coordinator_profile) }
+  let(:induction_coordinator_profile) { create(:induction_coordinator_profile, schools: [ect_profile.school]) }
```

Comment:
Why this change?

Reply:
yeah definitely not needed, i left this in.